### PR TITLE
Updating SDK addresses for xDai support

### PR DESCRIPTION
**Pull request by crisog**
The current SDK used on Honeyswap.org is published by me, this PR has the intention to publish the package under @1Hive org.

Package name can be changed, right now it's called uniswap-xdai-sdk

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@uniswap/sdk",
+  "name": "uniswap-xdai-sdk",
```
Comment (onbjerg) [inline package.json]:
```suggestion
  "name": "@1hive/honeyswap-sdk",
```

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
   "license": "MIT",
-  "version": "2.0.6",
+  "version": "3.0.2a3",
```
Comment (onbjerg) [inline package.json]:
```suggestion
  "version": "1.0.0",
```

**Review by onbjerg — CHANGES_REQUESTED** [review]
I'd change the package name and version

**Comment by onbjerg** [comment]
Hey @crisog, so sorry for the extremely delayed response. Is this the version of the library that Honeyswap is currently using? If so, I'd just merge it.

**Comment by lkngtn** [comment]
@crisog can this be closed?

**Comment by crisog** [comment]
> @crisog can this be closed?

I'm gonna merge this since this is the version being used right now.

**Pull request merged**